flow/project.py

```diff
@@ -3280,11 +3720,15 @@ def hello(job):
             raise ValueError("A group with name '{}' is already registered.".format(name))
 
         signature = inspect.signature(func)
+        total_params = len(signature.parameters)
         for i, (k, v) in enumerate(signature.parameters.items()):
-            if i and v.default is inspect.Parameter.empty:
+            if (
+                i != total_params - 1 and
+                v.kind is inspect.Parameter.VAR_POSITIONAL
+            ):
                 raise ValueError(
-                    "Only the first argument in an operation argument may not have "
-                    "a default value! ({})".format(name))
+                    "The variable parameter '{}' in the operation '{}' should be "
+                    "the last parameter!".format(k, name))
```

Comment:
This is not technically true they could have a function `foo(a, b, *c, d=False)` and this is a valid signature that would work. We will never use the keyword only parameter `d`, but the user internally could.

Reply:
I think then there's no point limiting users by raising some kind of error. I'll remove the parameter check.